config/celery.py

```diff
@@ -12,3 +13,7 @@
 # So all tasks in any corgi.tasks submodule are automatically discovered
 # And no config changes are needed when a new submodule is added
 app.autodiscover_tasks()
+
+trace.LOG_SUCCESS = """\
+%(return_value)s\" task_name=%(name)s, task_id=%(id)s, task_runtime=%(runtime)ss\
```

Comment:
Logging the runtime is a great idea, thanks for doing the research here!

Should we also log the same info on trace.LOG_FAILURE? And do you thik it's valuable to log the task args and task kwargs as well? I'm thinking of something like below:

```
LOG_FORMAT = """\
%(return_value)s\", task_name=%(name)s, task_id=%(id)s, task_args=%(args)s, task_kwargs=%(kwargs)s, task_runtime=%(runtime)ss\
"""

trace.LOG_SUCCESS = LOG_FORMAT
trace.LOG_FAILURE = LOG_FORMAT
```

Reply:
It's a good idea. Let's add it in a follow up tasks though.